lisa/wa.py

```diff
@@ -461,9 +461,11 @@ def _get_job_df(cls, job):
                 'metric': metric.name,
                 'value': metric.value,
                 'unit': metric.units or '',
+                'classifiers': dict(metric.classifiers),
```

Comment:
wouldn't this give the same result and avoid the json_normalize() post processing ?
```python
**metric.classifiers
```